sdk/keyvault/azure-security-keyvault-keys/src/cryptography/key_sign_parameters.cpp

```diff
@@ -1,9 +1,9 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include "azure/keyvault/keys/cryptography/key_sign_parameters.hpp"
-#include "azure/keyvault/keys/cryptography/cryptography_serializers.hpp"
-#include "azure/keyvault/keys/details/key_constants.hpp"
+#include "../private/key_sign_parameters.hpp"
```

Comment:
I don't think we use `#include "../` ever. Maybe let's add `src` to the list of include paths for when compiling `.cpp` (not for public), so that we can write `#include "private/..."`? The main thing to test if we do that would be to do Cmake install and make sure it completes and that it does not put the private headers in the `out/install/include` directory. If that's too complicated, using `../` should be fine, at least for now. But I believe we do it as I described in core already, so maybe see what we do in the CMakeFile there.

Reply:
We do that for unit test, but for the lib I didn't want to do it because it could bring confusion about not finding those header within the inc/ folder as all others.